classes/fields/datetime.php

```diff
@@ -281,8 +281,8 @@ public function input( $name, $value = null, $options = null, $pod = null, $id =
 			$value = implode( ' ', $value );
 		}
 
-		// Format Value
-		$value = $this->format_value_display( $value, $options, true );
+		// @todo Remove? Format Value (done in field template).
+		//$value = $this->format_value_display( $value, $options, true );
```

Comment:
:no_entry_sign: No space found before comment text; expected "// $value = $this->format_value_display( $value, $options, true );" but found "//$value = $this->format_value_display( $value, $options, true );"

<sub>hash: f504df84b70fa76d6ae61dc2461578d1</sub>